resources/player_override.ts

```diff
@@ -12,14 +15,18 @@ import Util from './util';
 // jobs remotely due to gauge data being local and many bits of information
 // loaded from memory.
 
-export const addPlayerChangedOverrideListener = function(playerName, func) {
+type PlayerChangedDetail = { detail: PlayerChangedRet };
+type PlayerChangedFunc = (e: PlayerChangedDetail) => void;
+
+export const addPlayerChangedOverrideListener = (playerName: string,
+    func: PlayerChangedFunc): void => {
   if (!func)
     return;
 
-  let lastPlayerChangedEvent = null;
-  let lastPlayerJob = null;
+  let lastPlayerChangedEvent: PlayerChangedDetail | null = null;
+  let lastPlayerJob: (Job | null) = null;
```

Comment:
Why does this one have parentheses? Any reason?

Reply:
No idea, fixed.